packages/compass-editor/src/codemirror/ace-compat-autocompleter.ts

```diff
@@ -21,6 +21,7 @@ export function mapMongoDBCompletionToCodemirrorCompletion(
         ? completion.value
         : wrapField(completion.value, escape === 'always'),
     detail: completion.meta?.startsWith('field') ? 'field' : completion.meta,
+    type: 'base-autocompleter',
```

Comment:
Adding this will start showing the rectangle box icon for all of our possible autocompletion types (`field`, `query`, `bson`, etc). This is shown in the screenshot. Should we instead show different icons for these different types? Maybe something similar to what VSCode does? https://code.visualstudio.com/docs/editor/intellisense#_types-of-completions 

Alternatively, we could hide icons for now and do that as a separate pr. It likely won't be too much so I think I'd prefer if we do it here. 

From a quick look I'm thinking right now we only need to add a new the regular box, which we can use for the functions, which would be `BSON` and `query`. Then we use the existing `base-autocompleter`/`field` rectangle box for the field autocompletions. Maybe there are more icons we can use for now? Not sure.
We get those non-field autocompletions from here: https://github.com/mongodb-js/devtools-shared/tree/main/packages/mongodb-constants 
Maybe some of the json-schema completions are `keyword`s.